contracts/src/protocol/OmniStake.sol

```diff
@@ -23,7 +23,8 @@ contract OmniStake {
     function deposit(bytes memory pubkey) external payable {
         require(msg.value >= 1 ether, "OmniStake: deposit amt too low");
         require(msg.value < type(uint64).max, "OmniStake: deposit amt too high");
-        require(msg.sender == Secp256k1.pubkeyToAddress(pubkey), "OmniStake: pubkey not sender");
+        // TODO(corver): Figure out another way to test failed evm log in cosmos.
```

Comment:
this contract is going to be refactors in the coming weeks. This is just a temporary workaround in order to test failed EVM logs. The refactored contract will also support failed testing.